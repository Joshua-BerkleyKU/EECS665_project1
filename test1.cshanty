plus 2
// this is a comment
"this is a valid string"
"this string is unterminated
"this string has bad escape \a"
