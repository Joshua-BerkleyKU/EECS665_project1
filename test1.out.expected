PLUS [1,1]
INTLIT:2 [1,6]
EOF [2,1]
